packages/react-mutation-mapper/src/component/mutationMapper/MutationMapper.tsx

```diff
@@ -239,6 +240,13 @@ export default class MutationMapper<
             : getDefaultWindowInstance();
     }
 
+    @computed
+    get usingOncoKbPublicInstance() {
+        return this.store.oncoKbInfo.result
+            ? this.store.oncoKbInfo.result.publicInstance
+            : ONCOKB_DEFAULT_INFO_PUBLIC_INSTANCE;
```

Comment:
Isn't it possible here to just do:
```suggestion
        return this.store.usingOncoKbPublicInstance;
```

Reply:
Oh good catch. This is must been there before I wrote the method in store

Reply:
i feel like it's odd to have something in all caps that is not a string or number constant.

Reply:
Hmm, I never thought about restricting all caps to string and number constants. We already have similar default values in `react-mutation-mapper` though.

Reply:
According to airbnb javascript style guideline, it seems okay to have all caps for other types of constants as well: https://github.com/airbnb/javascript/#naming--uppercase

Reply:
Oh, `ONCOKB_DEFAULT_INFO_PUBLIC_INSTANCE` is a boolean variable: https://github.com/cBioPortal/cbioportal-frontend/pull/3182/files#diff-c27e93a0aa919b85e6be54dc7efb1c85R33
Having the name `instance` for a boolean variable is a bit confusing here.